geniza/corpus/views.py

```diff
@@ -175,14 +178,20 @@ def get_queryset(self):
                         method="unified",
                         requireFieldMatch=True,
                     )
-                    # return smaller chunk of highlighted text for transcriptions
+                    # return smaller chunk of highlighted text for transcriptions/translations
                     # since the lines are often shorter, resulting in longer text
                     .highlight(
                         "transcription",
                         method="unified",
                         fragsize=150,  # try including more context
                         requireFieldMatch=True,
                     )
+                    .highlight(
+                        "translation",
+                        method="unified",
+                        fragsize=150,
+                        requireFieldMatch=True,
+                    )
```

Comment:
I guess you have to call highlight twice the way it's currently implemented. We might want to update parasolr with support for multiple highlight fields and the same common options (esp. if you're going to need multiple translation fields). You could define highlight params once and pass in as keyword args if they are the same, but for only two it's probably easier to keep them readable.

Reply:
Solr will take a list of fields or even a wildcard (which you'd probably want if you have multiple translation fields by language), but it will probably take a little bit of effort for parasolr to handle, since right now we set highlight parameters per field only.